manifest.xml

```diff
@@ -3,7 +3,7 @@
 	<name>Scheduling</name>
 	<id>scheduling</id>
 	<author>Juho Jaakkola</author>
-	<version>1.9.2</version>
+	<version>1.9.2-1</version>
```

Comment:
FYI: this is not a valid version number. The numbering must follow Semver: https://semver.org/

Also, the number should not be incremented within a fix PR . There may also be other pull requests open at the same time, and the number shouldn't be incremented until all of them have been accepted, and everything within the repository (e.g. possible changelog, readme, etc) has been prepared for the release.

Reply:
If your dev branch is behind the stable one (in this case `master`), use `git rebase <branch name>`, to make it look like you've done your changes on top of the latest master version.

Reply:
Thanks for the tip, I will try it.